### PR TITLE
Express/serve

### PR DESCRIPTION
**Pull request by ugogo**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const path = require("path");
+const axios = require("axios");
+const express = require("express");
+
+const { static } = express;
+const frontDir = path.join(__dirname, "../src");
+
+const { PORT = 3000, SLACK_TOKEN } = process.env;
```
Comment (ugogo) [inline server/index.js]: @arkan the token should be here

File: src/app.js [hunk 2]
```diff
@@ -37,23 +26,17 @@
...
 
-  const userToken = getTokenInURL();
-
-  if (!userToken) {
-    $tokenErrorContainer.classList.remove('is-hidden');
-    return;
-  }
-
-  const fetchURL = `https://slack.com/api/users.list?token=${userToken}&pretty=1`;
+  const fetchURL = `http://localhost:3000/pirates`;
```
Comment (arkan) [inline src/app.js]: I think you can only use `const fetchURL = "/pirates";`

File: server/index.js (hunk 1 above)
Comment (arkan) [reply server/index.js]: Why don't you use `process.env.SLACK_TOKEN` here?

File: src/app.js (hunk 2 above)
Comment (ugogo) [reply src/app.js]: Oupsy, fixed

File: server/index.js (hunk 1 above)
Comment (ugogo) [reply server/index.js]: I am
`const { SLACK_TOKEN } = process.env;` is the same as `const { SLACK_TOKEN } = process.env.SLACK_TOKEN;`

Comment (arkan) [reply server/index.js]: ah okay :smile:

**Comment by arkan** [comment]
This looks all good - Have you tested it yet?

**Comment by ugogo** [comment]
@arkan Tested locally with `SLACK_TOKEN={MY-TOKEN} npm run start` ✅

**Pull request merged**